src/test/ui/issues/issue-22560.stderr

```diff
@@ -29,20 +51,15 @@ LL |             Sub;
    |             additional non-auto trait
    |             trait alias used in trait object type (additional use)
 
-error[E0191]: the value of the associated types `Output` (from the trait `std::ops::Add`), `Output` (from the trait `std::ops::Sub`) must be specified
+error[E0191]: the value of the associated types `Output` (from trait `std::ops::Add`), `Output` (from trait `std::ops::Sub`) must be specified
   --> $DIR/issue-22560.rs:3:13
    |
 LL |   type Test = dyn Add +
    |  _____________^
-   | |_____________|
-   | |
 LL | |
 LL | |
 LL | |             Sub;
-   | |               ^
-   | |_______________|
-   | |_______________associated type `Output` must be specified
-   |                 associated type `Output` must be specified
+   | |_______________^ associated types `Output`, `Output` must be specified
```

Comment:
This is the case I'm protecting against when checking there's only one trait bound.

Reply:
Related to https://github.com/rust-lang/rust/issues/60595